@here/olp-sdk-core/lib/utils/index.ts

```diff
@@ -25,7 +25,10 @@ export enum STATUS_CODES {
     CREATED = 201,
     NO_CONTENT = 204,
     NOT_FOUND = 404,
-    SERVICE_UNAVAIBLE = 503
+    TO_MANY_REQUESTS = 429,
+    SERVICE_UNAVAIBLE = 503,
+    SERVER_ERROR_CODES_FROM = 500,
+    SERVER_ERROR_CODES_TO = 599
```

Comment:
I mean with proper names and in proper order

```suggestion
    INTERNAL_SERVER_ERROR = 500,
    SERVICE_UNAVAIBLE = 503,
    NETWORK_CONNECT_TIMEOUT = 599
```

Reply:
updated, thank you